runtime/src/iree/hal/command_buffer.h

```diff
@@ -297,7 +297,7 @@ typedef uint8_t iree_hal_collective_kind_t;
 // Specifies the reduction operator of a collective reduction operation.
 enum iree_hal_collective_reduction_e {
   // Specifies that the reduction operation computes a sum (addition).
-  IREE_HAL_COLLECTIVE_REDUCTION_SUM = 0,
```

Comment:
good catch - make an `IREE_HAL_COLLECTIVE_REDUCTION_NONE = 0` to match the HALBase.td enum instead of starting at 1